docs/release_guide.md

```diff
@@ -93,6 +94,10 @@ It will build the CLI binaries, publish a new GitHub release,
 upload the packaged binaries and checksums as release assets, build and push Docker images for OSM and the demo to the
 [`openservicemesh` organization](https://hub.docker.com/u/openservicemesh) on Docker Hub, and publish the Helm chart to the repo hosted at https://openservicemesh.github.io/osm.
 
+To summarize, here is a diagram demonstrate the git branching during the release process.
+
+![](../img/osm-git-release.jpg)
```

Comment:
Personally, I find this diagram a bit overwhelming and hard to follow. This seems like a symptom that something in the text could be made more clear, in which case I'd rather clarify the text than supplement it with a diagram in the spirit of trying to keep this document as lean and easy to follow as we reasonably can.

I'm certainly open to keeping it if others find something like this helpful, but it does need at least one change:

> 3. merge patch to release branch

At this point, there have been no changes added to the patch branch. The changes from steps 4 and 5 can be made on the patch branch then a single PR can be made from the patch branch to the release branch.

Reply:
Discussed offline. Diagram removed for simplicity.

Reply:
Certainly not necessary to handle in this PR, but another thought I had was to add a diagram that shows only the end result of the commits/branches/tags, since that's what's most important. Exactly how that result is achieved (PRs from certain branches, cherry-picking, merging) isn't all that important and may change, but it's good to document at least one method documented with concrete steps. I don't think the equivalent of that is described in words anywhere and would be useful context to have while performing the steps.

Again, I don't mean to dictate for everybody what is or isn't helpful, but I think it's important to avoid ambiguity in this doc as much as possible.

@allenlsy could you at least add the diagram back in a comment on this thread so we can get some more feedback?